tests/test_html.py

```diff
@@ -65,6 +65,7 @@ def test_illegal_entities(self):
         self.assertEqual(replace_entities("x&#x2264;y"), "x\u2264y")
         self.assertEqual(replace_entities("x&#157;y"), "xy")
         self.assertEqual(replace_entities("x&#157;y", remove_illegal=False), "x&#157;y")
+        self.assertEqual(replace_entities("&#82179209091;"), "")
```

Comment:
I believe your fix covers it, but for the sake of test comprehension, I think it would be nice to cover an additional scenario:

```suggestion
        self.assertEqual(replace_entities("&#82179209091;"), "")
        self.assertEqual(replace_entities("&#82179209091;", remove_illegal=False), "&#82179209091;")
```